apps/sms/test/marionette/generators/thread.js

```diff
@@ -0,0 +1,54 @@
+function ThreadGenerator() {};
+
+ThreadGenerator.uniqueThreadId = 0;
+ThreadGenerator.uniqueMessageId = 0;
+
+ThreadGenerator.injectDependencies = function(client) {
+  client.contentScript.inject(
+    __dirname + '/../mocks/mock_test_storages.js'
+  );
+  client.contentScript.inject(
+    __dirname + '/../mocks/mock_navigator_moz_mobile_message.js'
+  );
+};
+
+ThreadGenerator.generate = function(numberOfMessages, messageType, participants) {
+  if (typeof numberOfMessages === 'undefined') {
```

Comment:
`Object.assign` is not in node 0.10 (not tested on 0.12). But lodash (already included in packages.json) [has it](https://lodash.com/docs#assign).

Changed.